R/check.R

```diff
@@ -341,11 +368,12 @@ check_numeric_timestep <- function(dates, date_var, timestep, exact = TRUE) {
 #' @return This function is used for its side effect of stopping if the check
 #' fails. If the check passes, the function returns invisibly.
 #' @family check
+#' @importFrom rlang abort
 check_timestep <- function(obs, date_var, timestep = "day", exact = TRUE,
                            check_nrow = TRUE) {
   obs <- coerce_dt(obs, required_cols = date_var, copy = FALSE)
   if (!is.Date(obs[[date_var]])) {
-    stop(date_var, " must be of class Date")
+    rlang::abort(paste0(date_var, " must be of class Date"))
```

Comment:
Probably wants a slgihtly better message here. Maybe

`sprintf("Column `%s` must be class `Date`.", date_var)`

?